source/upgrade/important-upgrade-notes.rst

```diff
@@ -21,6 +21,13 @@ Important Upgrade Notes
 +----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | If you’re upgrading from a version earlier than... | Then...                                                                                                                                                          |
 +====================================================+==================================================================================================================================================================+
+| v6.4                                               | A new schema migration system has been introduced. Before updating the app to this version, it is strongly recommended to have a database backup. The new        |
```

Comment:
Same question about "app" here -- what "app" is being referred to?

Reply:
Changed it to "server".